lib/gobstones/expectations/expectations_runner.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../with_tempfile'
+
+module Gobstones::Expectations
+  class ExpectationsRunner
+    include Gobstones::WithTempfile
+
+    def initialize(gobstones_command, source_code)
+      @gobstones_command = gobstones_command
+      @source_code = source_code
```

Comment:
Could be, but that conflicts with
https://github.com/uqbar-project/gobstones-spec/issues/2

El jue, may 28, 2015 19:55, Franco Leonardo Bulgarelli <
notifications@github.com> escribió:

> In lib/gobstones/expectations/expectations_runner.rb
> <https://github.com/uqbar-project/gobstones-spec/pull/9#discussion_r31286080>
> :
>
> > @@ -0,0 +1,23 @@
> > +require_relative '../with_tempfile'
> > +
> > +module Gobstones::Expectations
> > +  class ExpectationsRunner
> > +    include Gobstones::WithTempfile
> > +
> > +    def initialize(gobstones_command, source_code)
> > +      @gobstones_command = gobstones_command
> > +      @source_code = source_code
>
> Perhaps you could use here
> Gobstones::Language::Gobstones.gobstones_commands
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uqbar-project/gobstones-spec/pull/9/files#r31286080>.
>